docfx-utils.ps1

```diff
@@ -0,0 +1,55 @@
+
```

Comment:
Why is there a `docfx-util.ps1` and a `docfx-utils.ps1` file?

Reply:
oh i meant to remove docfx-util.ps1. I kept making the mistake of typing `docfx-utils.ps1` so I figure I should go with what's more natural